.github/workflows/Issue Templates/bug.yml

```diff
@@ -0,0 +1,44 @@
+name: Bug Report🐛
+description: Create a report to help us improve
+title: "Bug🐛: "
+labels: [bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to file a Bug report
+  - type: textarea
+    attributes:
+      label: Describe the bug.
+      description: "Enter a clear and concise description of what the bug is"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the steps to reproduce the behavior
+      description: "Be clear and concise while describing the steps to ensure that it can be reproduced and potentially fixed"
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
```

Comment:
node is a cli based application,
For placeholder we can request for version `assetNode version`, followed by the the cli command used by the user e.g `assetClient keys add [key_name]`.

We should keep version or git commit hash requirement as true `required: true`